modules/core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/repository/Repository.java

```diff
@@ -45,6 +45,9 @@
     @XNode("@isDefault")
     private Boolean isDefault;
 
+    @XNode("@headless")
```

Comment:
Do we need the `@XNode` here as this descriptor is only kept for backward-compatibility? Nobody will contribute with this attribute.

Reply:
we don't, no. I didn't have it initially but added for consistency, will gladly remove it